assets/js/stripe.js

```diff
@@ -1,6 +1,12 @@
 /* global wc_stripe_params, Stripe */
 
-jQuery( function( $ ) {
+import {
+	PAYMENT_METHOD_BOLETO,
+	PAYMENT_METHOD_OXXO,
+	PAYMENT_METHOD_SEPA,
+} from 'wcstripe/stripe-utils/constants';
```

Comment:
@wjrosa I am getting errors for legacy shortcode checkout from this, with the message `Uncaught SyntaxError: Cannot use import statement outside a module`.